src/herder/TransactionQueue.h

```diff
@@ -170,7 +172,6 @@ class TransactionQueue
      */
     using BannedTransactions = std::deque<UnorderedSet<Hash>>;
 
-    Application& mApp;
```

Comment:
Glad to see mApp removed! Seems like it's actually not too painful to get rid of it, please correct me if I'm wrong: it was mostly used to grab the LCL from LedgerManager, which is now done via mValidationSnapshot (and hopefully later can be done with just the mBucketSnapshot object). For the rest of the functionality, we use mAppConnector which either requires calling certain methods from the main thread, and forces users to implement thread-safe functions. From what I can tell, besides the timestamp getter, no new thread-safe methods needed to be introduced, which is definitely good news.

Reply:
That's exactly right!